src/somef/parser/pom_xml_parser.py

```diff
@@ -0,0 +1,259 @@
+        with open(file_path, 'rb') as file:
+            data = lxml.etree.parse(file)
+            
+            root = data.getroot()
+            if root.tag != "{" + POM_NAMESPACE + "}project":
+                logging.warning(f"Expected root tag 'project' in {POM_NAMESPACE} namespace, got {root.tag} instead")
+                return metadata_result
+
+            group_id = None
+            artifact_id = None
+            version_value = None
+
+            project_data = {
+               "identifier": None,
+                "version": None,
+                "name": None,
+                "description": None,
+                "url": None,
+                "licenses": [],
+                "dependencies": [],
+                "developers": [],
+                "issue_tracker": None,
+                "scm_url": None
+            }
+
+            for key, node in parse_node(root):
+                if key == "groupId":
+                    group_id = node.text
+                elif key == "artifactId":
+                    artifact_id = node.text
+                elif key == "version":
+                    version_value = node.text
+                elif key == "name":
+                    project_data["name"] = node.text
+                elif key == "licenses":
+                    project_data["licenses"] = parse_licenses(node)
+                # elif key == "repositories":
+                #     repositories = parse_repositories(node)
+                elif key == "issueManagement":
+                    project_data["issue_tracker"] = parse_issue_management(node)
+                elif key == "scm":
+                    project_data["scm_url"] = parse_scm(node)        
+                elif key == "developers":
+                    project_data["developers"] = parse_developers(node)              
+                elif key == "dependencies":
+                    project_data["dependencies"] = parse_dependencies(node)
+            
+            if group_id or artifact_id:
+                identifier_value = f"{group_id or ''}.{artifact_id or ''}".strip(':')
+                metadata_result.add_result(
+                    constants.CAT_IDENTIFIER,
+                    {
+                        "value": identifier_value,
+                        "type": "string"
+                    },
+                    1,
+                    constants.TECHNIQUE_CODE_CONFIG_PARSER,
+                    file_path
+                )
+
+            if project_data["name"]:
+                metadata_result.add_result(
+                    constants.CAT_NAME,
+                    {
+                        "value": project_data["name"],
+                        "type": "string"
+                    },
+                    1,
+                    constants.TECHNIQUE_CODE_CONFIG_PARSER,
+                    file_path
+                )
+
+            if version_value:
+                metadata_result.add_result(
+                    constants.CAT_RELEASES,

```

Comment:
If there is a version value, it should not go in releases, but version number, no?

Reply:
Maybe here we can track a new category which is latest version